tcf_website/views/review.py

```diff
@@ -114,7 +114,6 @@ class DeleteReview(LoginRequiredMixin, generic.DeleteView):
     """Review deletion view."""
     model = Review
     success_url = reverse_lazy('reviews')
-    success_message = "Successfully deleted the review for %(course)s"
```

Comment:
Sorry I missed this in the first commit. This is a vestige from when I tried to implement the message using `SuccessMessageMixin`, but I ended up overriding `delete` instead.